package.json

```diff
@@ -43,7 +43,6 @@
 		"connect-mongo": "5.0.0",
 		"cookie-parser": "1.4.6",
 		"cors": "2.8.5",
```

Comment:
discord is removed? @Enubia

Reply:
Yes the bot is gone already

Reply:
This doesn't affect the discord webhooks notifications, right? Just making sure

Reply:
Nope the webhooks are not affected by this